request-manager/spec/errors.go

```diff
@@ -43,7 +39,7 @@ func (e InvalidValueError) Error() string {
 		multipleValues = "s"
 	}
 
-	return fmt.Sprintf("%s: invalid value%s %s in field '%s', expected %s",
+	return fmt.Sprintf("%sinvalid value%s %s in field '%s', expected %s",
```

Comment:
Typo: no space between "%s" and "invalid"

Reply:
These are deliberate; either %s is the empty string, in which case we don't want a leading space, or it's "node: ", which comes with a space.